app/models/concerns/orderable.rb

```diff
@@ -2,9 +2,16 @@ module Orderable
   extend ActiveSupport::Concern
 
   included do
-    default_scope { 
-      order(featured: :desc, comments_count: :desc, created_at: :desc, id: :desc) 
+    scope :order_by, lambda{ |field|
```

Comment:
I'd probably call this something other than field – maybe `sorting_criteria` instead?